scm-record/src/types.rs

```diff
@@ -125,10 +125,14 @@ impl TryFrom<i32> for FileMode {
     }
 }
 
+/// The state of the selection.
 #[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 pub enum Tristate {
+    /// All elements are not selected.
     False,
+    /// Some elements are selected.
     Partial,
+    /// All elements are selected.
```

Comment:
Very willing to rework the wording here, with some guidance. It doesn't seem _bad_ as it is now, but I'm sure it could be a lot better :)

Reply:
Seems fine to me 🙂.